example/lib/main.dart

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// ignore_for_file: library_private_types_in_public_api
```

Comment:
??

Reply:
New lint rules warn in statefull widgets that is not okay define private class